RecoLocalCalo/HcalRecAlgos/src/HcalSimpleRecAlgo.cc

```diff
@@ -361,26 +354,26 @@ namespace HcalSimpleRecAlgoImpl {
 
     // Note that uncorr_ampl is always set from outside of method 2!
     if( puCorrMethod == 2 ){
-       std::vector<double> correctedOutput;
+      buffer.clear();
 
-       CaloSamples cs;
-       coder.adc2fC(digi,cs);
-       std::vector<int> capidvec;
+      CaloSamples cs;
+      coder.adc2fC(digi,cs);
+      std::vector<int> capidvec;
       for(int ip=0; ip<cs.size(); ip++){
         const int capid = digi[ip].capid();
         capidvec.push_back(capid);
       }
-      psFitOOTpuCorr->apply(cs, capidvec, calibs, correctedOutput);
-      if( correctedOutput.back() == 0 && correctedOutput.size() >1 ){
-	time = correctedOutput[1]; ampl = correctedOutput[0];
+      psFitOOTpuCorr->apply(cs, capidvec, calibs, buffer);
+      if( buffer.back() == 0 && buffer.size() >1 ){
```

Comment:
At this point, I don't really want to change the interface of two classes written by somebody else.